cloudinary/api.py

```diff
@@ -123,12 +123,44 @@ def resource(public_id, **options):
     resource_type = options.pop("resource_type", "image")
     upload_type = options.pop("type", "upload")
     uri = ["resources", resource_type, upload_type, public_id]
-    params = only(options, "exif", "faces", "colors", "image_metadata", "cinemagraph_analysis",
-                  "pages", "phash", "coordinates", "max_results", "quality_analysis", "derived_next_cursor",
-                  "accessibility_analysis", "versions")
+    params = prepare_asset_details_params(**options)
     return call_api("get", uri, params, **options)
 
 
+def resource_by_asset_id(asset_id, **options):
+    """
+    Returns the details of the specified asset and all its derived assets by asset id.
+
+    Note that if you only need details about the original asset,
+    you can also use the Uploader::upload or Uploader::explicit methods, which return the same information and
```

Comment:
`Uploader::upload` and `Uploader::explicit` are not python methods, please fix.

You can see an example of a python docstring here:
https://github.com/cloudinary/pycloudinary/blob/86948bb280fc27906fff5ac81800bbb9d1908751/cloudinary/api.py#L96